### PR TITLE
add affiliate program overview

### PR DESCRIPTION
**Pull request by chefrabbid**

### CODE REVIEW

File: docs/pages/contracts/affiliate-program/overview.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+description: Fork of Curve Finance
```
Comment (ChefMist) [inline docs/pages/contracts/affiliate-program/overview.md:2]: in this description intended for Affiliate program?

Comment (ChefMist) [reply docs/pages/contracts/affiliate-program/overview.md:2]: Screenshot 2024-04-24 at 2 38 32 PM

this is how the page intended, is it expected to have `Stable Swap` keyword too?

**Review by ChefMist — COMMENTED** [review]
looks good overall, just curious about the headers

File: docs/pages/contracts/affiliate-program/overview.md (hunk 1 above)
Comment (chefrabbid) [reply docs/pages/contracts/affiliate-program/overview.md]: got it.

**Review by ChefMist — APPROVED** [review]

**Pull request merged**